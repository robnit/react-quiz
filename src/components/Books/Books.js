import React from 'react'

const Books = () => (
  <h1>BOOKS</h1>
)

export default Books