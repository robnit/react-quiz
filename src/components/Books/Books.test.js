import React from 'react'
import Books from 'components/Books/Books'
import { mount } from 'enzyme' 

describe('book', () => {
  it('should match snapshots', () => {
    const wrapper = mount(<Books/>)
   
    expect(wrapper).toMatchSnapshot()
  })
})
