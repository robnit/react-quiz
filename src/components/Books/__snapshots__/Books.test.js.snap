// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`book should match snapshots 1`] = `
<Books>
  <AddBooks>
    <h1>
      AddBooks
    </h1>
  </AddBooks>
  <Book>
    <h1>
      BOOKS
    </h1>
  </Book>
</Books>
`;
